src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionUITests.swift

```diff
@@ -277,108 +267,10 @@ class ENAUITests_04_ExposureSubmissionUITests: XCTestCase {
 		XCTAssertTrue(app.navigationBars["ENA.ExposureSubmissionWarnOthersView"].waitForExistence(timeout: .medium))
 		app.buttons["AppStrings.ExposureSubmission.continueText"].tap()
 
-	#if INTEROP
```

Comment:
Hmm shouldn't we remove the `INTEROP` from the Test targets? And shouldn't it everything in between ```
#if INTEROP
#endif
``` be the part we keep?

Reply:
These were UI tests that tested the old interoperability screens that have been scrubbed, so I had to remove the screens as well as the tests.

Reply:
Ah okay. Thanks!